### PR TITLE
Updated User Docs for new visibility modifiers.

### PR DESCRIPTION
**Pull request by jfdm**

### CODE REVIEW

File: docs/tutorial/modules.rst [hunk 1]
```diff
@@ -75,56 +75,91 @@ main module, with the ``main`` function, must be called
...
+- ``export`` the interface name is exported
+- ``public export`` the interface name, method names and default
+  definitions are exported
+
+
+``BTree`` Revisited
++++++++++++++++++++
+
+For our ``BTree`` module, it makes sense for the tree data type and the
 functions to be exported as ``abstract``, as we see below:
```
Comment (Melvar) [inline docs/tutorial/modules.rst]: Was this `abstract` missed or intentionally left in?

File: docs/tutorial/modules.rst [hunk 2]
```diff
@@ -151,7 +186,10 @@ directive, for example:
     toTree (x :: xs) = insert x (toTree xs)
 
 In this case, any function with no access modifier will be exported as
-``abstract``, rather than left ``private``.
+``export``, rather than left ``private``.
```
Comment (Melvar) [inline docs/tutorial/modules.rst]: Is `private` the default now, or was the thing with switching on the presence of *any* access modifier left in?

**Comment by jfdm** [comment]
cheers @Melvar in my haste I forgot a few things

File: docs/tutorial/modules.rst (hunk 2 above)
Comment (jfdm) [inline docs/tutorial/modules.rst]: From the mailing list post i gather that private is the default now.

**Pull request merged**